kato/kato-azure/kato-azure.gradle

```diff
@@ -0,0 +1,14 @@
+dependencies {
```

Comment:
I'm not sure there is value in `kato-azure`, kato as a "concept" should be disappearing.

I believe it makes the most sense to just add this into clouddriver-azure.